### DISCUSSION TITLE
Stop reconnecting at startup

### DISCUSSION THREAD

**Discussion by Mizar01** [dbeaver/dbeaver#8736] [Q&A] [answered]
I work with 6 to 7 db per session and all from different sources, sometimes with vpn networks, tunneling and so on.

One of the most annoying things of dbeaver is that when I start it, it tries to reconnect to all those databases, forcing me to close every error notice one by one and after that close all those broken connections.

I tried everything, all the options in the preferences, already suggested many times by users, but, nevertheless, it tries to connect. I definetely don't want that.

It seems like a bug to me, because is going on for months. 

The 'Disconnect All' menu is good, and in general it works, but I have to remember to click it before closing the app. But even this button sometimes does really nothing, and it leaves clearly the connection open (you can see it because the project tree remains opened on the connected DBs)  

Version in use: DBeaver v7.0.3 (Windows 10)

**Comment by uslss** [comment, marked as answer]
Hi @Mizar01 ,
you could disable it in Preferences
![image](https://user-images.githubusercontent.com/31996417/88383346-01b99100-cdb3-11ea-8a7e-ce6ff297f8b5.png)
![image](https://user-images.githubusercontent.com/31996417/88383381-14cc6100-cdb3-11ea-8e1f-34ac2cbef4d0.png)
